File: Bars.Gkh/Entities/Dicts/KindEquipment.cs
namespace Bars.Gkh.Entities
{
    /// <summary>
    /// Вид оснащения
    /// </summary>
    public class KindEquipment : BaseGkhEntity
    {
        /// <summary>
        /// Наименование
        /// </summary>
        public virtual string Name { get; set; }

        /// <summary>
        /// Описание
        /// </summary>
        public virtual string Description { get; set; }

        /// <summary>
        /// Ед. измерения
        /// </summary>
        public virtual UnitMeasure UnitMeasure { get; set; }
    }
}

File: Bars.Gkh/libs/B4/model/Dict/KindEquipment.js
Ext.define('B4.model.dict.KindEquipment', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'KindEquipment'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'Name' },
        { name: 'Description' },
        { name: 'UnitMeasure', defaultValue: null }
    ]
});